ixofoundation/ixo-webclient
# chore(style): update search icons

## Scope
The All projects icon needs to be changed

## Work Done
Added icons using a new implementation that allows for easier svg manipulation

## Steps to Test
On the projects page click the search dropdown and make sure you can see all the icons 

## Screenshots
![image](https://user-images.githubusercontent.com/46312800/74947165-23c57680-5403-11ea-923f-7b3dfadbc296.png)
